test/specs/ReporterUtils.spec.js

```diff
@@ -88,15 +88,28 @@ describe("Testing ReporterUtils.js", () => {
 
         it("nominal", () => {
             const result = extractSpecFromSuite({
-                title: "My title [@spec: test/Sample.spec.js]"
+                title: 'My title [@spec: {\"relative\":\"test/Sample.spec.js\",\"absolute\":\"/builds/group/project/test/Sample.spec.js\"}]'
+            }, {
+                useAbsoluteSpecPath: false
             });
             expect(result).toBe("test/Sample.spec.js");
         });
 
-        it("without spec in title", () => {
+        it("with absolute spec file option", () => {
+            const result = extractSpecFromSuite({
+                title: "My title [@spec: {\"relative\":\"test/Sample.spec.js\",\"absolute\":\"/builds/group/project/test/Sample.spec.js\"}]"
+            }, {
+                useAbsoluteSpecPath: true
+            });
+            expect(result).toBe("/builds/group/project/test/Sample.spec.js");
```

Comment:
This assert is still Linux style dependent (doesn't work under Windows).
And the hard coded path doesn't work on all machines.